spec/bowling_spec.rb

```diff
@@ -0,0 +1,179 @@
+require 'bowling'
```

Comment:
Think about your inputs and outputs with your tests. Start from the easiest final output (a gutter game) and build up to something more complex (a mixed game with consecutive spares and strikes). It's good to see a chronological logic to your code.